dispatcher/backend/src/tests/integration/routes/conftest.py

```diff
@@ -93,6 +96,31 @@ def schedule(make_schedule):
     return make_schedule()
 
 
+@pytest.fixture
+def temp_schedule(make_schedule):
```

Comment:
How is this specific? Tests data are not supposed to be persisted so all of them are _temporary_.

Reply:
Other test data are issued by a fixture with scope "module" ; this one has a scope "function", meaning it will be recreated for each test case (avoiding side-effects from other test cases since we are modifying the data quite heavily).

Reply:
OK, I guess it's helpful in this way.